License2Deploy/rolling_deploy.py

```diff
@@ -136,15 +137,12 @@ def get_instance_ids_by_requested_build_tag(self, id_list, build):
       new_instances += [instance_id for new_id in instances_build_tags if new_id.tags['BUILD'] == str(build)]
 
     if not new_instances:
-      logging.error("There are no instances in the group with build number {0}. Please ensure AMI was promoted.\nInstance ID List: {1}".format(build, id_list))
-      group_name = self.get_autoscale_group_name()
-      self.set_autoscale_instance_desired_count(self.calculate_autoscale_desired_instance_count(group_name, 'decrease'), group_name)
-      exit(self.exit_error_code)
-
-    id_ip_dict = self.get_instance_ip_addrs(new_instances)
-    logging.info("New Instance List with IP Addresses: {0}".format(id_ip_dict))
-    return new_instances
-
+      raise
```

Comment:
There is a lot of useful info above. We want to be transparent with why it is failing.